src/arguments.js

```diff
@@ -44,12 +44,12 @@ const unifyDescription = (description) => {
 const getDefaultValue = (description, regexp) => {
     let result = getValueByRegexp(description, regexp);
     if (result) {
-        result = removeCharAtBegin(result[5].trim(), ['\'', '\"', ',']);
-        result = removeCharAtTheEnd(result, ['\'', '\"', ',', '.']);
+        result = removeCharAtBegin(result[5].trim(), ['\'', '\"', ',', '[']);
```

Comment:
I suggest to create separate const for `regexp`

Reply:
@svoboda-rabstvo maybe you mean for an **array** with chars?